src/DetailsView/reports/components/report-sections/details-section.tsx

```diff
@@ -10,53 +12,29 @@ import { SectionProps } from './report-section-factory';
 
 export type DetailsSectionProps = Pick<SectionProps, 'pageUrl' | 'description' | 'scanDate' | 'toUtcString'>;
 
-const createCommentRow = (description: string, screenReaderText: string) => (
-    <li>
-        <span className="icon" aria-hidden="true">
-            <CommentIcon />
-        </span>
-        <span className="screen-reader-only" id="comment-text">
-            {screenReaderText}
-        </span>
-        <span className="text description-text" aria-labelledby="comment-text" aria-hidden="true">
-            {description}
-        </span>
-    </li>
-);
-
 export const DetailsSection = NamedSFC<DetailsSectionProps>('DetailsSection', props => {
     const { pageUrl, description, scanDate, toUtcString } = props;
+
+    const createListItem = (icon: JSX.Element, label: string, content: string | JSX.Element, contentClassName?: string) => (
+        <li>
+            <span className="icon" aria-hidden="true">
+                {icon}
+            </span>
+            <span className="screen-reader-only">{label}</span>
```

Comment:
This is going to read the content for everything with an extra down arrow press on screen-readers. Are we fine with that?

Reply:
I checked with both, JAWS and NVDA. Both AT's read date and comment altogheter (to my surprise)

Reply:
Awesome.